### PR TITLE
{data}[intel/2018a] GDAL v2.2.3 w/ Python 3.6.4

### PR DESCRIPTION
**Pull request by reedts**
(created using `eb --new-pr`)
Edit: depends on ~~#5437~~
Edit2: depends also on ~~#5405~~ and ~~#5439~~

### CODE REVIEW

File: easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    ('GEOS', '3.6.2', versionsuffix),
+    ('SQLite', '3.20.1'),
+    ('libxml2', '2.9.4'),
+    ('libpng', '1.6.32'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('JasPer', '2.0.12'),
+    ('LibTIFF', '4.0.9'),
+    ('zlib', '1.2.11'),
+    ('cURL', '7.55.1'),
+    ('PCRE', '8.41'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb]: @reedts existing `GDAL` easyconfigs also have a dependency on `PROJ` which is missing here?

Comment (reedts) [reply easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb]: @boegel Maybe it makes sense to bump this PR to `intel-2018a` as it is quite old. I would then update GDAL and its dependencies to the current versions...

Comment (boegel) [reply easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb]: @reedts Makes sense to me, the just opened #6022 could be useful here...

Comment (reedts) [reply easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb]: @boegel Thanks! I'm on it.

Comment (boegel) [reply easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018.01-Python-3.6.3.eb]: If we're not going to get this one merged, let's close it?

**Comment by reedts** [comment]
This now depends on #6012

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2706.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4f74b2d1321a6782f043cbe397d369dc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2036.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/16c5d7f8c0f33525243da039e0f70a6d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @reedts!

**Pull request merged**